### ISSUE TITLE
Use data from external blob

### ISSUE THREAD

**Issue by itmilos** [Azure-Samples/azure-search-openai-demo#225]
### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [] feature request
- [x ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

I would like to use data from external blob what would be the best way to do it with current soluton ? 

I don't want to download all files to local store them in data folder and then deploy them. 

I want to connect to blov and have all file splited and analysed.

**Comment by adrianoaraujocoelho** [comment]
Take a look at this link:
https://github.com/Azure-Samples/azure-search-openai-demo/pull/35

**Comment by itmilos** [comment]
Hi @adrianoaraujocoelho I did look at #35 but it is not working :(

**Comment by adrianoaraujocoelho** [comment]
did you download the code? What's the error?

**Comment by itmilos** [comment]
@adrianoaraujocoelho Yes I did it fails to upload document most likely due to authorization issue

**Comment by dGanguly1** [comment]
I have a sharepoint index set up in my cognitive search service in Azure. The index has data. I want to modify the code so that it can reference the sharepoint index instead of the default gptkbindex. Is there any way to do that?
